src/persist-client/tests/trace/compaction_apply_res

```diff
@@ -0,0 +1,133 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+push-batch
+[0][2][0] 1 k0
```

Comment:
nit: the names of these keys make it a little hard for me to follow. can we replace them with the k0-2 style names that always specify exactly the lower and upper?